src/platforms/javascript/guides/capacitor/troubleshooting.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve common issues with the Capacitor SDK."
```

Comment:
You might also add keywords to help with search, like this:
keywords: ["minimum version"]